plugins-it/kafka-it/src/test/java/com/navercorp/pinpoint/plugin/kafka/KafkaClientITBase.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020 NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.navercorp.pinpoint.plugin.kafka;
+
+import com.navercorp.pinpoint.bootstrap.plugin.test.ExpectedAnnotation;
+import com.navercorp.pinpoint.bootstrap.plugin.test.ExpectedTrace;
+import com.navercorp.pinpoint.bootstrap.plugin.test.PluginTestVerifier;
+import com.navercorp.pinpoint.bootstrap.plugin.test.PluginTestVerifierHolder;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.producer.Callback;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import test.pinpoint.plugin.kafka.*;
+
+import java.lang.reflect.Method;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import static com.navercorp.pinpoint.bootstrap.plugin.test.Expectations.annotation;
+import static com.navercorp.pinpoint.bootstrap.plugin.test.Expectations.event;
+import static test.pinpoint.plugin.kafka.KafkaITConstants.*;
+
+/**
+ * @author Younsung Hwang
+ */
+public abstract class KafkaClientITBase {
+
+    private static KafkaUnitServer kafkaUnitServer = new KafkaUnitServer(2189, 9092);
+    protected static final TestProducer producer = new TestProducer();
+    private static ExecutorService executorService = Executors.newSingleThreadExecutor();
```

Comment:
Using daemon thread in Test could cause a bad situation. 
Please change this thread to daemon. 

If you need any reference, please check following code. 
`com.navercorp.test.pinpointplugin.activemqPollingMessageReceiver`
or change ThreadFactory that generates daemon thread.

Reply:
@koo-taejin
As you said, it is appropriate to be daemon thread.
I changed the polling thread to the daemon thread.